contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseWindowAppendOutputOperator.java

```diff
@@ -63,50 +55,35 @@
  *            The tuple type
  * @since 1.0.2
  */
-public abstract class AbstractHBaseWindowAppendOutputOperator<T> extends AbstractAggregateTransactionableStoreOutputOperator<T, HBaseWindowStore> {
+public abstract class AbstractHBaseWindowAppendOutputOperator<T> extends AbstractHBaseWindowOutputOperator<T> {
   private static final transient Logger logger = LoggerFactory.getLogger(AbstractHBaseWindowAppendOutputOperator.class);
-  private List<T> tuples;
   private transient ProcessingMode mode;
+  
+  @Deprecated
   public ProcessingMode getMode()
   {
     return mode;
   }
 
+  @Deprecated
   public void setMode(ProcessingMode mode)
   {
     this.mode = mode;
   }
 
   public AbstractHBaseWindowAppendOutputOperator() {
     store = new HBaseWindowStore();
-    tuples = new ArrayList<T>();
   }
 
   @Override
-  public void storeAggregate() {
```

Comment:
Please see my description above. The window name is not for batching of tuples but rather for fault recovery behavior. The batching of tuples is not optimal as I described in the last paragraph above and hence I changed it. This also means the class hierarchy is slightly different but I don't think it will be a problem as I don't see anybody extending the class and overriding this method. Also since the operator is in contrib it is evolving in nature.